.github/issue_template.md

```diff
@@ -1,7 +1,17 @@
-Note: This is just a template, so feel free to use/remove the unnecessary things
+_Note:_ this is a template, please remove the parts that are not
```

Comment:
A lot of issue submitters keep the template and type their own stuff above or below. I don't think italicizing “Note” will improve that. Can we make this leading sentence punchier?